src/Cardiompp.Domain/Repositories/ICardiomppAgentRepository.cs

```diff
@@ -0,0 +1,10 @@
+﻿using Cardiompp.Domain.Entities;
+using System.Threading.Tasks;
+
+namespace Cardiompp.Domain.Repositories
+{
+    public interface ICardiomppAgentRepository
+    {
+        Task<CardiomppAgent> GetByNameAndPassword(string name, string password);
```

Comment:
Existe um convenção pra adicionar o suffixo `Async` aos métodos assícronos. 
```suggestion
        Task<CardiomppAgent> GetByNameAndPasswordAsync(string name, string password);
```

Reply:
Feito.  

![image](https://user-images.githubusercontent.com/12301754/80872582-2822d100-8c89-
11ea-935a-bc63a9a03a17.png)

Reply:
Feito.

![image](https://user-images.githubusercontent.com/12301754/80872595-340e9300-8c89-11ea-8138-79cd33a7df26.png)